### PR TITLE
Feature/unit sending

### PR DESCRIPTION
**Pull request by MakMuftic**
I just added the RPC method and waiting to test it with `Westend` before adding some UI inside the dashboard component.

### CODE REVIEW

File: packages/snap/src/polkadot/api.ts [hunk 1]
```diff
@@ -22,9 +23,11 @@ async function initApi(wsRpcUrl: string): Promise<ApiPromise> {
 }
 
 export const getApi = async (wallet: Wallet): Promise<ApiPromise> => {
-  if (!api) {
+  const config = getConfiguration(wallet);
+  // api not initialized or configuration changed
+  if (!api || rpcUrl != config.wsRpcUrl) {
```
Comment (MakMuftic) [inline packages/snap/src/polkadot/api.ts:28]: This is what I was referring to today that API wasn't reinitializing on network change.

Comment (mpetrunic) [reply packages/snap/src/polkadot/api.ts:28]: doesn't api already have some way to fetch rpc url. If not, it's better to just set api to null when configure method is called

Comment (mpetrunic) [reply packages/snap/src/polkadot/api.ts:28]: Generally open PR with just fix like that, because we know have to wait until this PR got merged to have that fixed. If possible, try to cherry pick commit with this fix or create another branch and pr with this fix.

File: packages/snap/src/rpc/substrate/sendUnit.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import ApiPromise from "@polkadot/api/promise";
+import {Wallet} from "../../interfaces";
+import { Keyring } from "@polkadot/api";
+import { Hash } from '@polkadot/types/interfaces/runtime';
+
+export async function sendUnit(wallet: Wallet, api: ApiPromise, amount: string|number, to: string): Promise<Hash> {
+  const keyring = new Keyring();
+  const keypair = keyring.addFromJson(wallet.getPluginState().polkadot.account.keyring);
```
Comment (mpetrunic) [inline packages/snap/src/rpc/substrate/sendUnit.ts:8]: Don't we already have some thod to get keypair?

File: packages/snap/src/rpc/substrate/sendUnit.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const { nonce } = await api.query.system.account(keypair.address);
```
Comment (mpetrunic) [inline packages/snap/src/rpc/substrate/sendUnit.ts:9]: https://polkadot.js.org/api/start/api.tx.html

According to this, sign and send will obtain next nonce by itself

File: packages/snap/src/rpc/substrate/sendUnit.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  const transfer = api.tx.balances.transfer(to, amount);
+  return new Promise((resolve) => {
+    transfer.signAndSend(keypair, {nonce}, ({ events = [], status }) => {
+      if (status.isInBlock) {
```
Comment (mpetrunic) [inline packages/snap/src/rpc/substrate/sendUnit.ts:13]: Maybe it's better to return hash, and expose exevnt listener for inBlock and finalized but we can do that in separate pr

Comment (MakMuftic) [reply packages/snap/src/rpc/substrate/sendUnit.ts:13]: I will refactor `sendUnit` function to return a hash for now and create issues for creating event listeners for inBlock and finalized.
